Project-3/Project#3_tf

```diff
@@ -0,0 +1,292 @@
+ 
+So, to modeling the expression:
+
+Sally is instance_of object (person, robot, mannequin...)
+Tuesday is instance_of one-dimensional temporal region 
+Wednesday is an instance of one-dimensional temporal region 
+Arm is an instance of fiat object part or an instance_of object (applies equally as well to person, robot, mannequin...)
+“Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+“Sally participates in having no arms on Wednesday” is an instance of occurrent
+“Sally participates in having at most one arm on Wednesday” is an instance of occurrent
+
+Putting aside the above remarks, my answer is:
+
+“Sally participates in having at least one arm on Tuesday and Sally participates in having at most one arm on Wednesday”***
+
+***I think this rendering captures best the logical ambiguity discussed at length in the first part of my answer. I am not sure which day precedes which, since we do not know that both days belong to the same week.
+
+
+  (b) Every liver has some cell as part at all times it exists.
+
+Liver is a class in Uberon (UBERON_0002107), which is a subclass of material entity. 
+https://ontobee.org/ontology/UBERON?iri=http://purl.obolibrary.org/obo/UBERON_0002107
+
+Argument according to authority BS for the object property has_part_at_all_times: https://youtu.be/fkkWkTIxrNQ
+
+Liver has_part_at_all_times Cell
+
+
+  (c) John was a child, then an adult, then a senior. 
+
+John is an instance_of object (person, etc)
+childhood is an instance_of occurrent.
+adulthood is an instance_of occurrent.
+seniorhood is an instance_of occurrent.
+
+“John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood.”
+
+n.b., it isn't clear that adulthood and seniorhood are mutually exclusive instances, and there may be some overlap between the two. 
+
+
+  (d) Goofus and Gallant are married at each point in a three year span. 
+
+Goofus is an instance_of object
+Gallant is an instance_of object
+Marriage is an instance_of occurrent
+“Three years span 1” is an instance_of one-dimensional temporal region.
+T1 is an instance_of zero-dimensional temporal region.
+If zero-dimensional temporal region t1 is part_of the one-dimensional temporal region “three years span 1”, then Goofus participates in marriage at t1 and Galland participates in marriage at t1.
+
+n.b., This expression doesn’t say anything about Goofus and Gallant being married to each other, so this is not represented.
```

Comment:
Ah but it should be; ambiguities suggest additional modeling is needed, not less.